build/windows/Makefile.ps1

```diff
@@ -5,7 +5,7 @@
 #  3. copy the files under installer directory to ..\..\kubernetes\windows\amalogswindows
 #  4. Builds the livenessprobe cpp and copy the executable to the under directory ..\..\kubernetes\windows\amalogswindows
 
-$dotnetcoreframework = "netcoreapp3.1"
+$dotnetcoreframework = "net7.0"
```

Comment:
Should we move to net6.0? That has long term support as compared to 7.0 https://dotnet.microsoft.com/en-us/platform/support/policy

Reply:
> Should we move to net6.0? That has long term support as compared to 7.0 https://dotnet.microsoft.com/en-us/platform/support/policy

Thanks for catching this. Make sense to go with net6.0 considering its LTS. updated.